hack/scripts/break_mig.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import atexit
+import collections
+import re
+import subprocess
+import sys
+import time
+
+
+InstanceInfo = collections.namedtuple("InstanceInfo", 'name ip')
+
+
```

Comment:
Added docstrings to methods and top-level docstring explaining what the script does